benchttp/benchttp.go

```diff
@@ -21,7 +21,18 @@ func (b Benchmark) Times() []float64 {
 	return s
 }
 
+// StatusCodes returns the recorded HTTP status codes of the
+// requests as a slice of ints.
+func (b Benchmark) StatusCodes() []int {
+	s := make([]int, len(b.Records))
+	for i, v := range b.Records {
+		s[i] = v.Code
+	}
+	return s
+}
```

Comment:
[suggestion] Avoid iterating twice on the (potentially very long) records slice:
```suggestion
// StatusCodes returns the recorded HTTP status codes of the
// requests as a slice of ints.
func (b Benchmark) Values() (httpcodes []int, times []float64) {
	httpcodes = make([]int, len(b.Records))
	times = make([]float64, len(b.Records))
	for i, v := range b.Records {
		httpCodes[i] = v.Code
		times[i] = v.Time
	}
	return
}
```
(+ remove `Benchmark.Times)`